sdk/go/common/util/logging/log.go

```diff
@@ -155,38 +167,84 @@ func AddGlobalFilter(filter Filter) {
 	rwLock.Unlock()
 }
 
-func CreateFilter(secrets []string, replacement string) Filter {
-	items := make([]string, 0, len(secrets))
+func CreateGlobalFilter(secrets []string, replacement string) {
```

Comment:
I think this could be helped with some documentation.
It would be helpful to have comments explaining:

- the fact that this will merge with prior global filters (related: consider calling this PushGlobalFilter or something similarly additive because "create" hints that it'll replace)
- the advantage of using this over `AddGlobalFilter(CreateFilter())`

Also, given that this appends, consider a private `resetGlobalFilter` function that nils it out so that unit tests that call CreateGlobalFilter can `defer resetGlobalFilter()`.